ctrl.c

```diff
@@ -796,6 +796,14 @@ static int pv_ctrl_check_command(int req_fd, struct pv_cmd **cmd)
 		goto error;
 	}
 
+	if (!pv_config_get_bool(PV_DEBUG_SHELL) &&
```

Comment:
I would expect another error message for when user is trying to defer but nothing can be done. That is, when timeout is not yet activated.